### PR TITLE
Mobile Overhaul: Text Styling

### PR DESCRIPTION
**Pull request by tpachmann**
This PR should fix / implement the issue #498 (Mobile Overhaul: Adjust heading sizes & text overflow).

The font size for headings has been generally decreased and various overflow (ellipsis) rules have been added.

### CODE REVIEW

File: app/src/styles/styles.scss [hunk 1]
```diff
@@ -107,3 +113,32 @@ hr {
...
+  display: inline-block;
+  max-width: 100%;
+  word-wrap: break-word;
+  hyphens: auto;
+}
+
+.title-label {
+  word-wrap: break-word;
+  hyphens: auto;
+}
```
Comment (DPhuongHda) [inline app/src/styles/styles.scss:144]: add newline

File: app/src/app/pages/data-protection/data-protection.component.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.container {
+    iframe {
+        width: 100%;
+        max-width: 600px;
+    }
+}
```
Comment (DPhuongHda) [inline app/src/app/pages/data-protection/data-protection.component.scss:6]: add newline

**Review by DPhuongHda — CHANGES_REQUESTED** [review]
Looks good :) , maybe just add the missing newlines in the css files and then the pr can be merged.

**Comment by tpachmann** [comment]
Done :) Just out of interest: Is this a convention in this repo or otherwise preferable?

**Comment by DPhuongHda** [comment]
> Done :) Just out of interest: Is this a convention in this repo or otherwise preferable?

It is a general convention. Github is also displaying a small message if a new line is missing at the end of a file (review page).

**Review by DPhuongHda — APPROVED** [review]

**Pull request merged**